frontend/src/scenes/notebooks/Notebook/Notebook.scss

```diff
@@ -73,6 +73,10 @@
         top: 0;
         z-index: 1;
     }
+
+    span::selection {
```

Comment:
```suggestion
    // overriding ::selection is necessary here because
    // antd makes it invisible otherwise
    span::selection {
```

Reply:
thinking its best to capture the context for the ::selection to make life easier for the future traveller